.travis.yml

```diff
@@ -13,7 +13,7 @@ git:
 services:
   - docker
 script:
-  - make ci e2e status=keep
+  - make ci e2e status=keep globalnet=true
```

Comment:
E2e with Globalnet build: https://travis-ci.com/submariner-io/submariner-operator/builds/151560337

Reply:
We should remove this check (i.e., globalnet=true) before merging this PR right?

Reply:
> We should remove this check (i.e., globalnet=true) before merging this PR right?

Yes.